String Manipulation/Character Removal/Python/character_removal.py

```diff
@@ -0,0 +1,16 @@
+"""
+    Remove all occurences of char to_remove from string s
+"""
+def remove_character(s, to_remove):    
+    result = ""
+    for c in s:
+        if c != to_remove:
+            result += c
```

Comment:
concatenating string frequently is not an efficient approach. Its better to use a list instead. Append to the list & then use `''.join` while returning the string.

&nbsp;&nbsp;`result = []`
&nbsp;&nbsp;`for c in s:`
&nbsp;&nbsp;&nbsp;&nbsp;`if c != to_remove:`
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`result.append(c)`
&nbsp;&nbsp;`return ''.join(result)`